packages/styles/stories/Components/quick-view/no-header.example.html

```diff
@@ -1,4 +1,4 @@
-<div class="fd-popover">
+<div class="fd-popover" style="margin-bottom: 300px;">
```

Comment:
Please add a wrapping div element with margin-bottom and not apply it to the Popover as we might confuse the end users. For example:
```
<div style="margin-bottom: 300px;">
    <div class="fd-popover">...</div>
</div>
```